crates/ruma-client-api/CHANGELOG.md

```diff
@@ -2,187 +2,190 @@
 
 Breaking changes:
 
-- `RoomSummary::heroes` now properly contains only `UserId` instead of `String`
-  as before.
-- Change type of `client_secret` field in `ThirdpartyIdCredentials`
-  from `Box<ClientSecret>` to `OwnedClientSecret`
-- Make `id_server` and `id_access_token` in `ThirdpartyIdCredentials` optional
+-   `RoomSummary::heroes` now properly contains only `UserId` instead of `String`
+    as before.
+-   Change type of `client_secret` field in `ThirdpartyIdCredentials`
+    from `Box<ClientSecret>` to `OwnedClientSecret`
+-   Make `id_server` and `id_access_token` in `ThirdpartyIdCredentials` optional
 
 Improvements:
 
-- Add support for MSC4108 OIDC sign in and E2EE set up via QR code
-- Heroes in `sync::sync_events::v4`: `SyncRequestList` and `RoomSubscription`
-  both have a new `include_heroes` field. `SlidingSyncRoom` has a new `heroes`
-  field, with a new type `SlidingSyncRoomHero`.
-- Add support for authenticated media endpoints, according to MSC3916 / Matrix
-  1.11.
-  - They replace the newly deprecated `media::get_*` endpoints.
-- Stabilize support for animated thumbnails, according to Matrix 1.11
-- Add support for terms of service at registration, according to MSC1692 /
-  Matrix 1.11
+-   Add support for MSC4108 OIDC sign in and E2EE set up via QR code
+-   Heroes in `sync::sync_events::v4`: `SyncRequestList` and `RoomSubscription`
+    both have a new `include_heroes` field. `SlidingSyncRoom` has a new `heroes`
+    field, with a new type `SlidingSyncRoomHero`.
+-   Add support for authenticated media endpoints, according to MSC3916 / Matrix
+    1.11.
+    -   They replace the newly deprecated `media::get_*` endpoints.
+-   Stabilize support for animated thumbnails, according to Matrix 1.11
+-   Add support for terms of service at registration, according to MSC1692 /
+    Matrix 1.11
+-   Add support for [MSC4140](https://github.com/matrix-org/matrix-spec-proposals/pull/4140) to send `Future` events and update `Future` events with `future_tokens`.
```

Comment:
```suggestion
-   Add unstable support for [MSC4140](https://github.com/matrix-org/matrix-spec-proposals/pull/4140) to send `Future` events and update `Future` events with `future_tokens`.
```

Also, could you wrap the line at around 80 characters like the rest of the file, and revert the formatting changes?